### PR TITLE
Contribution: Add "I have no idea where to start" section

### PR DESCRIPTION
**Pull request by EwoutH**
A section for people who would like to contribute but don't know where to start. It points to a lot of resources in a semi-structured way.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -36,6 +36,39 @@ discuss via [Matrix] OR via [an issue].
...
+- [`https://github.dev/projectmesa/mesa`](https://github.dev/projectmesa/mesa) is great for small changes (to docs).
+
+Learn the tools, talk to us about what you want to change, and open a small PR. Or update an [example model](https://github.com/projectmesa/mesa-examples) (check open [issues](https://github.com/projectmesa/mesa-examples/issues))!
+
+### I'm a developer (but not a modeller)
+Awesome! You have the basics of open-source software development (if not check above), but not much modelling experience.
+
+First step is to start thinking like a modeller. To understand the fine details about our library and contribute meaningfully, get some modelling experience:
+- Go though our [Introductory Tutorial](https://mesa.readthedocs.io/en/latest/tutorials/intro_tutorial.html) and [Visualization Tutorial](https://mesa.readthedocs.io/en/latest/tutorials/visualization_tutorial.html). While going through them, dive into the source code to really see what everything does.
+- Follow an ABM course (if possible).
```
Comment (rht) [inline CONTRIBUTING.md:57]: Needs to add @tpike3's MOOC complexity tutorial course on Sugarscape. https://www.complexityexplorer.org/courses/172-agent-based-models-with-python-an-introduction-to-mesa. The problem is that the Mesa version needed is very dated.

**Review by rht — APPROVED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (EwoutH) [reply CONTRIBUTING.md:57]: Done!

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -36,6 +36,40 @@ discuss via [Matrix] OR via [an issue].
...
+  - This MOOC on ABM concepts: [Agent Based Modeling](https://ocw.tudelft.nl/course-lectures/agent-based-modeling/)
+  - This MOOC on practical ABM modelling: [Agent-Based Models with Python: An Introduction to Mesa](https://www.complexityexplorer.org/courses/172-agent-based-models-with-python-an-introduction-to-mesa)
```
Comment (rht) [inline CONTRIBUTING.md:59]: Should add a warning that the Mesa version used there is currently too old. <2.3.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -36,6 +36,40 @@ discuss via [Matrix] OR via [an issue].
...
+
+Learn the tools, talk to us about what you want to change, and open a small PR. Or update an [example model](https://github.com/projectmesa/mesa-examples) (check open [issues](https://github.com/projectmesa/mesa-examples/issues))!
+
+### I'm a developer (but not a modeller)
+Awesome! You have the basics of open-source software development (if not check above), but not much modelling experience.
+
+First step is to start thinking like a modeller. To understand the fine details about our library and contribute meaningfully, get some modelling experience:
+- Go though our [Introductory Tutorial](https://mesa.readthedocs.io/en/latest/tutorials/intro_tutorial.html) and [Visualization Tutorial](https://mesa.readthedocs.io/en/latest/tutorials/visualization_tutorial.html). While going through them, dive into the source code to really see what everything does.
+- Follow an ABM course (if possible).
+  - This MOOC on ABM concepts: [Agent Based Modeling](https://ocw.tudelft.nl/course-lectures/agent-based-modeling/)
```
Comment (rht) [inline CONTRIBUTING.md:58]: I just checked that this is based on NetLogo. People looking for Mesa are probably people who prefer Python over an obscure language. Is there a Python-based course?

Comment (EwoutH) [reply CONTRIBUTING.md:58]: Not as far as I know. But the ABM concepts are the same right?

Calling NetLogo obscure in the ABM world is also quite a statement.

File: CONTRIBUTING.md (hunk 2 above)
Comment (EwoutH) [reply CONTRIBUTING.md:59]: Then we can better remove it right?

**Comment by EwoutH** [comment]
Thanks for merging!

(this one if fine, but regularly I would like to merge myself. If it's open I might want some more eyes on it, make some minor changes, or merge in a specific way)

**Pull request merged**